x-pack/plugins/security_solution_serverless/server/cloud_security/cloud_security_metering.ts

```diff
@@ -25,20 +26,31 @@ export const cloudSecurityMetringCallback = async ({
   const tier: Tier = getCloudProductTier(config, logger);
 
   try {
-    const cloudSecuritySolutions: CloudSecuritySolutions[] = [CSPM, KSPM, CNVM, CLOUD_DEFEND];
+    const cloudSecuritySolutions: CloudSecuritySolutions[] = [CLOUD_DEFEND];
 
     const promiseResults = await Promise.allSettled(
```

Comment:
It might be good to additionally return a [shouldRunAgain and latestTimestamp](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution_serverless/server/types.ts#L101) along with the records so that if there are more heartbeat documents, the task will run again immediately to pick up the rest of the heartbeats.

Reply:
Great input!

[Here's](https://github.com/elastic/kibana/pull/1%20v83814/files#diff-610588098e2e4b244a535e1c937edf268a59b85dabb741f071a6c0ea80a211cfR131) how I addressed it.

Reply:
Your approach makes sense to me so I went ahead and approved. Something to keep in mind with this approach is that if there is a very large amount of heartbeats the task might potentially run into the 1 minute timeout.